docs/tutorial.md

```diff
@@ -150,25 +149,36 @@ commit ID, which changes when you rewrite the commit. You can give either ID
 to commands that take revisions as arguments. We will generally prefer change
 IDs because they stay the same when the commit is rewritten.
 
-By default, `jj log` lists your local commits, with some remote commits added
-for context.  The `~` indicates that the commit has parents that are not
-included in the graph. We can use the `--revisions`/`-r` flag to select a
-different set of revisions to list. The flag accepts a ["revset"](revsets.md),
-which is an expression in a simple language for specifying revisions. For
-example, `@` refers to the working-copy commit, `root()` refers to the root
-commit, `branches()` refers to all commits pointed to by branches. We can
-combine expressions with `|` for union, `&` for intersection and `~` for
-difference. For example:
+By default, `jj log` lists your local commits.  The `~` indicates that the
+commit has parents that are not included in the graph. We can use the
+`--revisions`/`-r` flag to select a different set of revisions to list. The flag
+accepts a ["revset"](revsets.md), which is an expression in a simple language
+for specifying revisions. For example, `@` refers to the working-copy commit,
+`root()` refers to the root commit, `trunk()` refers to the repo's main branch
+(`master` in this case, as determined by `jj git clone`), and `trunk()..` refers
+to all commits that are not ancestors of the main branch. We can combine
+expressions with `|` for union, `&` for intersection and `~` for difference. For
+example:
 
 ```shell
-$ jj log -r '@ | root() | branches()'
+$ jj log -r 'root() | trunk() | trunk().. | @'
```

Comment:
I'm wondering whether a larger change to the tutorial is needed now that `jj log` does not list remote branches by default, and now that the remote branches are immutable by default. It seems important to explain `jj log -r all()` and perhaps how to make the remote branch commits mutable. In fact, I'm not sure what is the minimum amount of info to explain to help people not get confused.

I'm considering removing the first commit from this PR until we get a better idea here.